playbooks/aws/openshift-master/config.yml

```diff
@@ -30,11 +31,13 @@
   vars_files:
     - vars.yml
   roles:
-    - ../../../roles/base_os
-    - ../../../roles/repos
+    - base_os
+    - repos
     - {
-        role: ../../../roles/openshift_master,
-        oo_node_ips: "{{ hostvars['localhost'].oo_node_ips | default(['']) }}",
-        oo_bind_ip: "{{ hostvars[inventory_hostname].ansible_eth0.ipv4.address | default(['']) }}"
+        role: openshift,
+        openshift_roles: ['master'],
```

Comment:
Agreed, I'd still like to keep as much of the duplicated/similar config contained within a single role. Any objections to either thin wrappers that set role specific type config around the generic openshift role, or having a common role that is pulled in as a dependency to the openshift-master and openshift-node roles?